src/DashboardService.Mappers/Db/DbCommentMapper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using LT.DigitalOffice.DashboardService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Comment;
+using Microsoft.AspNetCore.Http.HttpResults;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Db;
+
+public class DbCommentMapper : IDbCommentMapper
+{
+  public DbComment Map(CreateCommentRequest request)
+  {
+    return request is null
+      ? null
+      : new()
+      {
+        Id = Guid.NewGuid(),
+        TaskId = request.TaskId,
+        CreatedBy = request.CommentCreatedBy,
+        CreatedAtUtc = request.CommentCreatedAtUtc,
```

Comment:
this is not sent in the request, this is set already when handling the request: createdBy is set using httpContextAccessor, CreatedAtUtc - current utc date

Reply:
Use DateTime.Now

Reply:
*DateTime.UtcNow